pkg/proc/locations.go

```diff
@@ -0,0 +1,112 @@
+package proc
```

Comment:
If these are to be moved they should be moved to breakpoints.go since that's what they are related to.

Reply:
Better alternative: bininfo.go, since that's where other code related to the sourceline/address mapping is there.

It's weird to have a locations.go file that isn't related to the Location type and contains a bunch of functions that neither accept nor return Location objects.